static/src/stylesheets/module/email/_form.scss

```diff
@@ -0,0 +1,207 @@
+// TODO These mixins are here while we run the A/B test as the styles are custom to the email form but if they become perm then
+// they should be made reusable
```

Comment:
:eyes: don't forget about this! 

it's important we don't end up with a mishmash of style whims and their attendant css